### ISSUE TITLE
Package icon into jar and use it

### ISSUE THREAD

**Issue by FRosner** [FRosner/DataGenerator#205]
The icon should be included as a resource and packaged into the jar. It would be nice to have it replace the jar icon as well as a favicon in the frame.

---

Depends on: #174

**Comment by Husterknupp** [comment]
The current favicon is surrounded by a white frame, which should be removed.

**Comment by Husterknupp** [comment]
All further changes concerning the icon will be made in #174.

**Comment by FRosner** [comment]
I'll commit the logo here.
